src/components/Login/LoginFormJsonFile/LoginFormJsonFile.js

```diff
@@ -0,0 +1,65 @@
+
+import React from 'react';
+import { bool, string, func } from 'prop-types'; import { noop } from 'lodash';
+
+import ReactFileReader from 'react-file-reader'
+import Button from '../../Forms/Button';
+import styles from './LoginFormJsonFile.scss';
+
+export default class LoginFormJsonFile extends React.Component {
+  static propTypes = {
+    disabled: bool,
+    wif: string,
+    setWIF: func,
+    onLogin: func
+  };
+
+
+  static defaultProps = {
+    disabled: false,
+    wif: '',
+    setWIF: noop,
+    onLogin: noop
+  };
+
+
+handleFiles = files => {
+    const { setWIF } = this.props;
+
+    var reader = new FileReader();
+    reader.onload = e =>
+    {
+    let keyStoreJSON = JSON.parse(reader.result);

```

Comment:
We can actually just let neon-js load the account data from file.  See here: https://github.com/CityOfZion/neon-js/blob/a0967658535ac70361b749e9ba56742c3b2a5766/src/wallet/Wallet.js#L96

There's also an [`accountToNEP6`](https://github.com/nos/client/blob/242b71ca9f70eea23c40e7c799ab41e2f4914c91/src/util/accountToNEP6.js) file in this codebase, it would be nice to include an `accountFromNEP6` counterpart.

Reply:
We also need to handle the error condition here for an invalid file structure.